packages/utils/src/TestingUtils/JestUtils/TestWrapper.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+const TestWrapper = ({ renderOptions = {}, children }) => {
+  const mockStore = configureStore();
+
+  return (
+    <IntlProvider locale="en">
+      <Provider store={renderOptions?.store || mockStore()}>
```

Comment:
Any reason why the context props are nested in the `renderOptions` object?

Reply:
There is no firm intention. This made this component as dynamic as possible so we do not have to pass all the options. Rather according to rendering needs, we can pass render options and the component renders declaratively. I am open to any improvements, feel free to share your ideas :)

Reply:
> This made this component as dynamic as possible so we do not have to pass all the options

Not sure if I follow. Props can be optional. I know its a nitpick, but I'd just make the props flat. I can understand props nesting when a specific prop encompasses props if a child component and you just want to spread them, but this seems unnecessary.

Reply:
I think Martin's comment makes sense: we can extract `store`, `initialEntries`, some other keys to separate props for TestWrapper. This way we don't need to look up for `renderOptions` first to find the needed values.

Reply:
We actually have the props more "plain" in Inventory: https://github.com/RedHatInsights/insights-inventory-frontend/blob/e2123605436df6b8c84de8c39711d0be2a5a142a/src/Utilities/TestingUtilities.js#L9